langchain4j-google-ai-gemini/src/test/java/dev/langchain4j/model/googleai/GoogleAiGeminiChatModelIT.java

```diff
@@ -43,7 +43,7 @@
 
 public class GoogleAiGeminiChatModelIT {
 
-    private static final String GOOGLE_AI_GEMINI_API_KEY = System.getenv("GOOGLE_AI_GEMINI_API_KEY");
+    private static final String GOOGLE_AI_GEMINI_API_KEY = System.getenv("GEMINI_AI_KEY");
```

Comment:
Please keep it `GOOGLE_AI_GEMINI_API_KEY` or let's change it in CI builds as well

Reply:
Sorry, fixed!